Makefile

```diff
@@ -30,6 +30,7 @@ $(ENV_FILE):
 
 $(IO_DIR)/%.sqlite: $(ENV_FILE)
 	docker run --rm -e DB_DIR=/io -v$(IO_DIR):/io --env-file $(ENV_FILE) --entrypoint python3 $(IMAGE) /server/tests/gen_sqlite_db.py
+	@touch $@
```

Comment:
The database is created wit `read-only` attributes, so `touch` fails.

Reply:
WDYM? It works fine for me.

Reply:
Is it about the owner user?

Reply:
How about now?

Reply:
It did! Many thanks!